docs/DeveloperGuide.md

```diff
@@ -943,6 +982,62 @@ The above sequence diagram omits details such as internal method calls to `GetBu
 The only difference between the negative and non-negative messages is that the negative message places the `-` symbol before the `$` so `-$xx.xx` whereas the non-negative message does not contain the `-` symbol
 </div>
 
+#### Summary Statistics
+
+**Overview**
+
+The `summary` command creates a pop-up summary window that contains summary plots to provide some basic analytics about
+the user's expense data. 
+
+The activity diagram of the `summary` command is given below:
+
+<img src="images/unicash/SummaryActivityDiagram.png" width="700" />
```

Comment:
I believe you have to label the `else` branch of the "[user deletes all expenses]` as well